.meta

```diff
@@ -0,0 +1,35 @@
+{
+  "projects": {
+    "dagpenger-events": "git@github.com:navikt/dagpenger-events",
+    "dagpenger-funksjonelle-tester": "git@github.com:navikt/funksjonelle-tester.git",
+    "dagpenger-iac": "git@github.com:navikt/dagpenger-iac.git",
+    "dagpenger-joark-mottak": "git@github.com:navikt/joark-mottak.git",
+    "dagpenger-journalforing-ferdigstill": "git@github.com:navikt/journalforing-ferdigstill.git",
+    "dagpenger-oppslag": "git@github.com:navikt/dagpenger-oppslag.git",
+    "dagpenger-raspberry-monitor": "git@github.com:navikt/raspberry-monitor.git",
+    "dagpenger-regel-ui": "git@github.com:navikt/regel-ui.git",
```

Comment:
er der bevisst at det er `git@github.com:navikt/regel-ui.git` istedet for `git@github.com:navikt/dagpenger-regel-ui.git`, er det noe magi jeg ikke ser?

Reply:
samme gjelder regel-api, offset-monitor og flere ser det ut til

Reply:
Jeg som lagde meg en macro i vim for å konvertere fra XML til JSON som ikke tok høyde for alt :D